crates/opactl/src/main.rs

```diff
@@ -312,10 +321,28 @@ async fn dispatch_args<
         Some(("set-policy", matches)) => {
             let root_key: SigningKey = load_key_from_match("root-key", matches).into();
             let transactor_key: SigningKey = load_key_from_match("transactor-key", matches).into();
-            let policy: &PathBuf = matches.get_one("policy").unwrap();
-            let mut policy_file = File::open(policy)?;
-            let mut policy = vec![];
-            policy_file.read_to_end(&mut policy)?;
+            let policy: &String = matches.get_one("policy").unwrap();
+
+            enum UrlOrFile {
+                Url(Url),
+                File(PathBuf),
+            }
+
+            let policy = match policy.parse::<Url>() {
```

Comment:
I'm pretty sure `file://bundle.tar.gz` type urls will parse successfully here.

Reply:
And they are absolute too, resolved and documented